test/patient.test.js

```diff
@@ -0,0 +1,185 @@
+/* 
```

Comment:
Can we do a `Shift + Alt +. F` on this file. And all of the other files as well?

Reply:
Ideally, we should be following the  `Given_When_Then`/ `Arrange_Act_Assert` convention when naming unit tests. See the other tests for examples of this to fix the naming convention. This applies for all the unit tests you have.